docs/developer/website-development.mdx

```diff
@@ -12,3 +20,320 @@ For more elaborate changes, including adding new pages, you will need to clone t
+
+To ensure proper imporation in Webpack 5 (there are other ways, by adding magic after the filename, in Webpack 4), in `docusaurus.config.ts` we have this small plugin:
+
+```ts
+plugins: [
+() => ({
+    name: 'read-lf-source-code-files',
+    configureWebpack: () => ({
+    module: {
+        rules: [
+        {
+            // Any LF file that lies in path where some directory is named "codes"
+            test: /codes?\/.*\.lf$/,
+            type: "asset/source",
+        },
+        ],
+    },
+    })
+}),
+],
+```
+
+With this, if you use the statement like `import C_Schedule from '../assets/code/c/src/Schedule.lf';` you will get a TS string which Webpack loads when building. Then, you could use [`NoSelectorTargetCodeBlock`](#noselectortargetcodeblock) to show a bunch of lf code.
+
+:::info
+Remember to use relative import! If you don't, you will mess up the versioning.
+:::
+
+#### Example with NoSelectorTargetCodeBlock
+
+```tsx
+import C_HelloWorld from '../assets/code/c/src/HelloWorld.lf';
+import Cpp_HelloWorld from '../assets/code/cpp/src/HelloWorld.lf';
+import Py_HelloWorld from '../assets/code/py/src/HelloWorld.lf';
+import Rs_HelloWorld from '../assets/code/rs/src/HelloWorld.lf';
+import TS_HelloWorld from '../assets/code/ts/src/HelloWorld.lf';
+
+<NoSelectorTargetCodeBlock c={C_HelloWorld} cpp={Cpp_HelloWorld} py={Py_HelloWorld} rs={Rs_HelloWorld} ts={TS_HelloWorld} lf />
+```
+
+:::note[Rendered as]
+
+import C_HelloWorld from '../assets/code/c/src/HelloWorld.lf';
+import Cpp_HelloWorld from '../assets/code/cpp/src/HelloWorld.lf';
+import Py_HelloWorld from '../assets/code/py/src/HelloWorld.lf';
+import Rs_HelloWorld from '../assets/code/rs/src/HelloWorld.lf';
+import TS_HelloWorld from '../assets/code/ts/src/HelloWorld.lf';
+
+<NoSelectorTargetCodeBlock c={C_HelloWorld} cpp={Cpp_HelloWorld} py={Py_HelloWorld} rs={Rs_HelloWorld} ts={TS_HelloWorld} lf />
+
+:::
+
+#### DynamicMultiTargetCodeblock
+
+:::danger
+This is not working properly with versioning and will have issues with Webpack performance!
+:::
+
+:::danger
+This is not working properly now! Need to fix the file path in `content` in `DynamicMultiTargetCodeblock.tsx`.
+:::
+
+If you are **really** lazy, there is a utility which I wrote, `DynamicMultiTargetCodeblock`, that allows you to only supply a LF filename and all files will be dynamically imported. The issue is Webpack is not intelligent enough so if you use it, it will probably load all LF files in `assets`. Also, because relative import appears to be impossible, using it will always get you the latest code examples. See `src/components/LinguaFrancaMultiTargetUtils/DynamicMultiTargetCodeblock.tsx`.
+
+##### Examples
+
+```tsx
+<DynamicMultiTargetCodeblock c cpp ts rs py file="HelloWorld" />
+```
+
+To suppress warning, add 
+
+```tsx
+<DynamicMultiTargetCodeblock doNotTransformInMDX c cpp ts rs py file="HelloWorld" />
+```
+
+#### DynamicMultiTargetCodeblock with TransformDynamicLFFileImportToStatic
```

Comment:
Like previous comment. Move to comments in tsx file?

Reply:
OK, done